sqrl-tools/sqrl-cli/src/main/java/com/datasqrl/cmd/AbstractCompilerCommand.java

```diff
@@ -83,9 +86,12 @@ public void execute(ErrorCollector errors, String[] profiles, Path snapshotPath,
     }
 
     if (goal == ExecutionGoal.TEST) {
-      List<String> defaultKeys = List.of(EngineKeys.TEST, EngineKeys.DATABASE, EngineKeys.LOG,
-          EngineKeys.SERVER, EngineKeys.STREAMS);
-      sqrlConfig.setPipeline(defaultKeys);
+      Set<String> enabledEngines = new HashSet<>(sqrlConfig.getEnabledEngines());
+      enabledEngines.add(EngineKeys.TEST);
```

Comment:
This seems problematic. The test command only supports the engines that we have explicitly included. If the user has configured some other engine it will fail.

Reply:
The problem is that then the sqrl script can't be compiled because it's missing engines (if annotated with iceberg for example).

Reply:
Let's move that to a separate ticket then